### DISCUSSION TITLE
How to add more links as downloads progress?

### DISCUSSION THREAD

**Discussion by miguelarios** [Tzahi12345/YoutubeDL-Material#499] [Q&A] [answered]
Curious as how I can add more links to download. I add one link and I get concerned with removing the link  to add another as I think it will cancel the download.

**Comment by GlassedSilver** [comment, marked as answer]
If you're on the latest NIGHTLY then you can basically slam it as hard as you wish, especially so if you're using the mongoDB backend :)

**Reply by GlassedSilver** [reply]
I'll consider this solved then since there hasn't been any follow-up. If I'm mistaken, let me know. If you have any other questions, don't hesitate to start a new thread or for bugs/enhancement ideas an issue. :)
